dlang.org.ddoc

```diff
@@ -205,6 +205,7 @@ MENU = <li><a href='$1'><span>$+</span></a></li>
 MENU_W_SUBMENU = <li class='expand-container'><a class='expand-toggle' href='#'><span>$0</span></a>
 MENU_W_SUBMENU_LINK = <li class='expand-container'><a class='expand-toggle' href='$1'><span>$+</span></a>
 MENU_W_SUBMENU_END = </li>
+MESSAGE_BOX = $(DIVC message-box message-box-$1, $+)
```

Comment:
We might define `RED_MESSAGE_BOX` or `ERROR_MESSAGE_BOX` etc. in the future, but for now I kept it as bloat-free as possible.